### PR TITLE
lang: add Dutch (NL) translation

### PR DESCRIPTION
**Pull request by pe7er**
Added  Dutch (Netherlands) language strings to languages-nl.json

### CODE REVIEW

**Comment by pe7er** [comment]
done!

File: lang/languages-nl.json [hunk 1]
```diff
@@ -1,27 +1,38 @@
...
\ No newline at end of file
+    "en": "Engels",
+    "af": "Afrikaans",
+    "az": "Azerbeidzjaans",
+    "bg": "Bulgaars",
+    "ca": "Catalaans",
+    "cs": "Tsjechisch",
+    "da": "Deens",
+    "de": "Duits",
+    "enGB": "English (Verenigd Koninkrijk)",
```
Comment (rubenk) [inline lang/languages-nl.json:10]: Engels I presume?

Comment (pe7er) [reply lang/languages-nl.json:10]: Oops, inderdaad. :-)

Comment (pe7er) [reply lang/languages-nl.json:10]: Would you like to fix it, or should I submit a patch?

Comment (rubenk) [reply lang/languages-nl.json:10]: I'm no Jitsi maintainer, just some Dutch guy trying to help ;-)
So please update this PR.

Comment (pe7er) [reply lang/languages-nl.json]: This other Dutch guy thanks you for that!
I've pushed the fix.

**Pull request merged**